app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,56 @@
+module ManagerRefresh::Inventory::Core
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def provider_module
+      @provider_module ||= ManageIQ::Providers::Inflector.provider_module(self)
+    end
+
+    def has_authentications(options = {})
```

Comment:
so can we put these definitions into a modules? So I can add the other managers and it's not all in one pile.

That being said, having it under one class will have a problem with conflicts. E.g. InfraManager vms vs. CloudManager vms, so that is partially the reason I went with the definitions I have now.